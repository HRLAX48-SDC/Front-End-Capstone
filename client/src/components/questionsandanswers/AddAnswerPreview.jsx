import React from 'react';
import { Image } from 'react-bootstrap';

const AddAnswerPreview = (props) => (
  <div className='q_previews'>
    {props.photos.map((photo, i) =>
      <Image src={photo} key={i} height='100' width='100'/>
    )}
  </div>
);

export default AddAnswerPreview;