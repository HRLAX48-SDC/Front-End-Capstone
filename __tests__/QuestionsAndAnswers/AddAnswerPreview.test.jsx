import React from 'react';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import AddAnswerPreview from '../../client/src/components/questionsandanswers/AddAnswerPreview.jsx';
Enzyme.configure({ adapter: new Adapter()});

const photos = ['http://res.cloudinary.com/dtnikmimx/image/upload/v1641450048/t8kuusgqlzx4x0qpigrv.jpg', 'http://res.cloudinary.com/dtnikmimx/image/upload/v1641450048/t8kuusgqlzx4x0qpigrv.jpg'];
describe('<AddAnswerPreview />', () => {
  test('render correctly', () => {
    const preview = renderer.create(<AddAnswerPreview photos={photos} />).toJSON();
    expect(preview).toMatchSnapshot();
  });
});