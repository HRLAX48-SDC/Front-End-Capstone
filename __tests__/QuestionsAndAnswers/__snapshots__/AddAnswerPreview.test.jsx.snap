// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddAnswerPreview /> render correctly 1`] = `
<div
  className="q_previews"
>
  <img
    className=""
    height="100"
    src="http://res.cloudinary.com/dtnikmimx/image/upload/v1641450048/t8kuusgqlzx4x0qpigrv.jpg"
    width="100"
  />
  <img
    className=""
    height="100"
    src="http://res.cloudinary.com/dtnikmimx/image/upload/v1641450048/t8kuusgqlzx4x0qpigrv.jpg"
    width="100"
  />
</div>
`;
